hq/app/controllers/HQController.scala

```diff
@@ -23,8 +24,13 @@ class HQController(val config: Configuration)
     Ok(views.html.index(accounts))
   }
 
-  def iam = AuthAction {
-    Ok(views.html.iam.iam())
+  def iam = AuthAction.async  {
+    attempt {
+      val accountsAndReports = IAMClient.getAllCredentialReports(accounts)
+      for {
+        accountReportMaps <- Attempt.sequence(accountsAndReports.toList)
+       } yield Ok(views.html.iam.iam(accountReportMaps.toMap))
```

Comment:
When running this locally and going to the IAM page, I'm getting an error:

`Unknown error while making an API call to AWS' AmazonIdentityManagement service`

I assume because I have only obtained the creds for a couple of accounts from Janus (and the others have expired). If this is the case, this won't be an issue in PROD (hopefully). However, maybe we could use a similar approach to the security-groups (which still gives you some results) to improve the DEV experience?

Reply:
Additionally, would it be worth adding a custom error at this point to help Devs if they run into this?

Reply:
It's because Janus credentials can't make IAM calls. It's super annoying, but to run this feature locally you'd need to temporarily make permanent credentials to authenticate those calls.

You're right it's probably worth providing a good error for this that describes the above. That might be a bit fiddly to add (because it's you only get a generic error and this is for a specific problem) but definitely doable.

Reply:
we don't have this issue on PROD, it's happening because of multiple accounts configured in the local `~/.gu/security-hq.local` file. When I leave have only one account like below I managed to see some data. @adamnfish have you got any idea how to fix this one locally ?

```{   
 name = "security"
 id   = "security-test"
 roleArn = "dev-local"
}